dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/createDataSource.vue

```diff
@@ -465,7 +494,7 @@
 
     mounted () {
     },
-    components: { mPopup, mListBoxF }
+    components: { mPopup, mListBoxF, mSelectInput }
```

Comment:
Hi,
What is the purpose of introducing this component?
If you want to add drop-down filtering options, Ans-UI native is supported.
![image](https://user-images.githubusercontent.com/29545877/86549748-02f55c00-bf73-11ea-94e7-fc93020be7fa.png)

Reply:
Sorry, this is my question. Thank you very much for your code review.